### ISSUE TITLE
Gradio online app

### ISSUE THREAD

**Issue by tonigi** [MPI-Dortmund/pymissense#8]
Just an idea: apparently it can be made into an online Gradio app to be hosted e.g. on HuggingFace.

**Comment by thorstenwagner** [comment]
Good idea! When I'm back from vacation I gonna look into it. Or are you interested in making an PR?

06.10.2023 11:58:16 Toni G ***@***.***>:
